packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -28,7 +31,20 @@ enum MaterialListType {
   twoLine,
 
   /// A list tile that contains three lines of text.
-  threeLine
+  threeLine,
+}
+
+/// Defines the title font used for [ListTile] descendants of a [ListTileTheme].
+///
+/// List tiles that appear in a [Drawer] use the theme's [TextTheme.body2]
+/// text style, which is a little smaller than the theme's [TextTheme.subhead]
+/// text style, which is used by default.
+enum ListTileStyle {
+  // Use a title font that's appropriate for a normal [ListTile].
+  normal,
```

Comment:
Excellent point, "list" is more consistent. List and drawer are (generically speaking) containers for list tiles.